observability/otel/otel-forwarding-logs.adoc

```diff
@@ -0,0 +1,172 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="otel-forwarding-logs"]
+= Forwarding Logs to a LokiStack instance
+include::_attributes/common-attributes.adoc[]
+:context: otel-forwarding-logs
+
+To configure forwarding logs to a LokiStack instance, you can deploy and configure the OpenTelemetry Collector. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {loki-op} is installed. See _Additional resources_ on how to install LokiStack.
+* A supported LokiStack instance is deployed on the cluster.
+
+TODO add dev-preview note and that the `lokipush` exporter will be removed in future release in favour of OTLPHTTP.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.Example ServiceAccount
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+----
+
+. Create a cluster role that gives collector service account permissions to push logs to the LokiStack application tenant. The cluster role has also permissions for `k8sattributes` processor.
++
+.Example ClusterRole
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector-logs-writer
+rules:
+ - apiGroups: ["loki.grafana.com"]
+   resourceNames: ["logs"]
+   resources: ["application"]
+   verbs: ["create"]
+ - apiGroups: [""]
+   resources: ["pods", "namespaces", "nodes"]
+   verbs: ["get", "watch", "list"]
+ - apiGroups: ["apps"]
+   resources: ["replicasets"]
+   verbs: ["get", "list", "watch"]
```

Comment:
Need this additional permissions:
```
 - apiGroups: ["extensions"]
   resources: ["replicasets"]
   verbs: ["get", "list", "watch"]
```

Reply:
isn't this for old k8s clusters? 

https://www.pulumi.com/registry/packages/kubernetes/api-docs/extensions/v1beta1/replicaset/

Reply:
I didn't add it and it seemed to work just fine

Reply:
It was in the processor doc so added it just in case it doesn't work on any particular OCP version. I'm currently testing on 4.16 and it seems to work well.

Reply:
I've added it.